vertx-pojo-mapper-common/src/main/java/de/braintags/io/vertx/pojomapper/impl/AbstractDataStore.java

```diff
@@ -60,11 +61,12 @@ public AbstractDataStore(Vertx vertx, JsonObject properties) {
     this.vertx = vertx;
     this.properties = properties;
     initSupportedKeyGenerators();
+    defaultQueryLimit = properties.getInteger("defaultQueryLimit", 500);
```

Comment:
replace expression "defaultQueryLimit" by well documented constant in IDataStore